jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/toolwindow/SqsWindowUI.kt

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.sqs.toolwindow
+
+import com.intellij.openapi.project.Project
+import com.intellij.ui.components.JBTabbedPane
+import software.aws.toolkits.jetbrains.services.sqs.Queue
+import software.aws.toolkits.resources.message
+import javax.swing.JPanel
+
+// Will add more parameters once window is populated
+class SqsWindowUI(private val project: Project, val queue: Queue) {
+    val mainPanel = JBTabbedPane().apply {
+        this.add(message("sqs.queue.message.sampling"), JPanel())
+        this.add(message("sqs.queue.send.message"), JPanel())
+    }
+
+    fun openMessage(): SqsWindowUI {
```

Comment:
This returning `SqsWindowUI` is kind of weird, it's like mixing a builder and a thing to select. instead of doing `showQueue(queue, SqsWindowUI(project, queue).openMessage())`, I think a more Kotliny way to do it is `showQueue(queue, SqsWindowUI(project, queue)).apply { openMessage()}`, then openMessage is just the set